packages/lesswrong/components/sunshineDashboard/RejectContentButton.tsx

```diff
@@ -48,8 +48,8 @@ export const RejectContentButton = ({contentWrapper, classes}: {
           <ReplayIcon className={classes.icon} onClick={unrejectContent}/>
         </LWTooltip>
     </span>}
-    {!content.rejected && content.authorIsUnreviewed && <span className={classes.button}>
-      <RejectedIcon className={classes.icon} onClick={() => setShowRejectionDialog(true)}/> <MetaInfo>Reject</MetaInfo>
+    {!content.rejected && content.authorIsUnreviewed && <span className={classes.button} onClick={() => setShowRejectionDialog(true)}>
```

Comment:
fix the bug where you could only reject a post by clicking on the icon, rather than also having the label be clickable.